dm/openapi/spec/dm.yaml

```diff
@@ -809,6 +811,151 @@ paths:
               schema:
                 $ref: "#/components/schemas/ErrorWithMessage"
 
+  /api/v1/task_configs/batch_import:
+    post:
+      tags:
+        - task_config
+      summary: "batch import task config"
+      operationId: "DMAPIBatchImportTaskConfig"
+      requestBody:
+        description: "request body"
+        content:
+          "application/json":
+            schema:
+              $ref: "#/components/schemas/BatchImportTaskConfigRequest"
+      responses:
+        "202":
+          description: "success"
+          content:
+            "application/json":
+              schema:
+                $ref: "#/components/schemas/BatchImportTaskConfigResponse"
+        "400":
+          description: "failed"
+          content:
+            "application/json":
+              schema:
+                $ref: "#/components/schemas/ErrorWithMessage"
+
+  /api/v1/task_configs:
```

Comment:
maybe  use this path pattern ? 🤔

* list/create `api/vi/tasks/configs`?
* batch_import `api/vi/tasks/configs/batch_import`
* delete/get/update `api/vi/tasks/{task-name}/configs` 

need suggestion for other reviewers

Reply:
My point:
list/create api/vi/tasks/configs?
batch_import api/vi/tasks/configs/import
delete/get/update api/vi/tasks/configs/{task-name}

Reply:
addressed in https://github.com/pingcap/tiflow/pull/3656/commits/f585588ef4a73b89f094349bb74229e33cc4d247 
ptal